prov/hook/trace/src/hook_trace.c

```diff
@@ -831,6 +837,101 @@ static struct fi_ops_tagged trace_tagged_ops = {
 };
 
 
+static int trace_setname(fid_t fid, void *addr, size_t addrlen)
+{
+	int ret = 0;
+	struct hook_fabric *myfabric = hook_to_fabric(fid);
+
+	ret = fi_setname(hook_to_hfid(fid), addr, addrlen);
+	TRACE_CM(ret, myfabric, (addr) ? (*(uint64_t *)addr) : 0);
```

Comment:
What if addrlen > sizeof(uint64_t)?

Reply:
Good catch, will fix the log of addr in all relevant cm ops. I thought the addr parameter is in the same type as in ep ops.

Reply:
Use ofi_straddr_log() to print the address in a readable format.  It only works with sockaddr, I think, but that may be the best we can do for now.